src/pages/jobs/[slug].astro

```diff
@@ -5,10 +5,11 @@ import freelancers from '../../data_files/freelancers.json';
 import { generateNameSlug } from '../../utils/utils.js';
 import { Image } from 'astro:assets';
 import type { ImageMetadata } from 'astro';
-import { getFlagEmoji, hasSprykerCertifications } from '../../utils/utils.js';
```

Comment:
Do we still need getFlagEmoji() in utils then?

Reply:
Correct, removed it with the new commit